### DISCUSSION TITLE
Exclamation mark not removed after reload.

### DISCUSSION THREAD

**Discussion by NickGrease** [uBlockOrigin/uBlock-issues#2564] [General] [answered]
![image](https://user-images.githubusercontent.com/854582/228576903-20a81bc7-3165-43b1-8d87-4d459b69eb4d.png)

~~First time seeing it, can't find any explanation.~~
A simple reload did not solve the issue in a Brave browser, but Ctrl-F5 did fix it.

**Comment by ghost** [comment]
https://github.com/uBlockOrigin/uBlock-issues/wiki/Toolbar-icon

New in 1.48 https://github.com/gorhill/uBlock/releases/tag/1.48.0

**Reply by NickGrease** [reply]
Thank you for the reply, I apologize for my laziness in not searching thru release notes. With that said, I might suggest a tooltip on hover. Additionally a simple reload *did not* solve the issue in a Brave browser, but Ctrl-F5 did fix it.

**Reply by ghost** [reply]
> ...a simple reload _did not_ solve the issue in a Brave browser, but Ctrl-F5 did fix it.

Interesting, can you share the address?

**Reply by NickGrease** [reply]
If I could reliably reproduce it I'd be happy to share the site, but it's only happened once after the initial post, and never again. It's the kind of site you definitely want uBlock for, no doubt there's a bunch of shenanigans going on in the background. Sorry I can't be more of a help, but this issue seems very isolated (multiple tabs on the same site, and only 1 had this issue) and currently no longer producible.

**Comment by gorhill** [comment, marked as answer]
I found a case I can reproduce: `https://www.telsu.fi/`. Forcing a reload does not reset the exclamation mark. Forcing a hard reload does work. I will investigate whether acting on a webNavigation event could work around the issue.

**Reply by gorhill** [reply]
https://github.com/gorhill/uBlock/commit/0b0915526817ce1b0dd2a32cfeaff3f1b647b737
